app/views/past_chancellors/_person.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="govuk-grid-column-one-quarter" role="listitem">
+  <%= render "govuk_publishing_components/components/heading", {
+    text: name,
+    heading_level: 3,
+    font_size: "s"
+  } %>
+  <% if info[:service].kind_of?(Array) %>
```

Comment:
In my past foreign secretaries page, I've removed this logic by just making them all arrays. I'd imagine we'd make it an array in the content item too, plus I'm not a fan of this kind of mixing types. What do you reckon about changing them all to be arrays?

Reply:
That sounds like a good idea.  I've made the same change here too.